examples/DEMO1-satellite.md

```diff
@@ -0,0 +1,141 @@
+**# DEMO 1: Satellite Images Red Team Operation
+[[Demo Home]](../demo/README.md)
+
+## Objective
+Change the label of a satellite image (true label: `stadium`) into `airplane`.
+
+## Threat Model
+- Attacker knowledge: API access to model
+- Security violation: integrity
+
+## Steps
+
+1. Start the Counterfit CLI.
+    ```
+    $ python examples/terminal/terminal.py 
+   
+                              __            _____ __
+      _________  __  ______  / /____  _____/ __(_) /_
+     / ___/ __ \/ / / / __ \/ __/ _ \/ ___/ /_/ / __/
+    / /__/ /_/ / /_/ / / / / /_/  __/ /  / __/ / /
+    \___/\____/\__,_/_/ /_/\__/\___/_/  /_/ /_/\__/
+
+                    Version: 1.1.0
+    ```
+
+2. What models are defined in our environment?
+    ```
+    counterfit> list targets 
+    ┏━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+    ┃ Name            ┃ Model Type ┃ Data Type ┃ Input Shape   ┃ # Samples ┃ Endpoint                                             ┃
+    ┡━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━╇━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+    │ creditfraud     │ blackbox   │ tabular   │ (30,)         │ 0         │ creditfraud/creditfraud_sklearn_pipeline.pkl         │
+    │ digits_blackbox │ blackbox   │ image     │ (1, 28, 28)   │ 0         │ digits_blackbox/mnist_sklearn_pipeline.pkl           │
+    │ digits_keras    │ keras      │ image     │ (28, 28, 1)   │ 0         │ digits_keras/mnist_model.h5                          │
```

Comment:
Since you fixed the bug of displaying correct model type for `digits_keras` target, so could you update this command output to reflect model dataype correctly for `digits_keras` target?